### DISCUSSION TITLE
Update image to the latest version.

### DISCUSSION THREAD

**Discussion by Digital-Media-Group** [open-webui/open-webui#7346] [General]
There is no specification of how to update the latest version of the image from "Compose". Is there any automatic method in place?

**Comment by mahenning** [comment]
For automatic updates I use watchtower (https://github.com/containrrr/watchtower). This is another container running, which looks once each day (configurable) for image updates for open-webui, and restarts the open-webui container with the new image, if available. You can also use [the cleanup option](https://containrrr.dev/watchtower/arguments/#cleanup) to automatically delete the old image. 
Make sure to have your data directory mounted to not lose your chats with the re-creation of the container.
